Documentation/Contributors/BuildGuide/README.md

```diff
@@ -22,13 +23,15 @@
    * Yes
       * Clone the cesium repo, e.g., `git clone git@github.com:AnalyticalGraphicsInc/cesium.git`
       * Make changes in a branch, e.g., `git checkout -b my-feature`.
+* For more about our workflow, see [github pull request workflows](http://cesiumjs.org/2013/10/08/GitHub-Pull-Request-Workflows/).
```

Comment:
This is a great link, but I think it belongs in the contributing guide: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines